app/Console/Commands/GenerateArticle.php

```diff
@@ -164,13 +161,16 @@ protected function collectWebsiteInfo()
         }
     }
 
+    /**
+     * @param string $websiteDomain
+     */
     protected function getNewWebsite($websiteDomain)
     {
         $websiteArray = [];
         $websiteArray['name'] = $this->ask("What is suitable name of the website. (Ideally name should uniquely identify website.)");
         $websiteArray['site_url'] = $websiteDomain;
 
-        if($this->confirm("It there and RSS feed of the website? (And you know that) y|N")) {
+        if ($this->confirm("It there and RSS feed of the website? (And you know that) y|N")) {
```

Comment:
Can you please change message

`It there and RSS feed of the website? (And you know that) y|N`
to
`Is there any RSS feed of the website? (And you know that) y|N`